src/ensembl/src/genome/genomeActions.ts

```diff
@@ -21,12 +28,30 @@ export const fetchGenomeInfoAsyncActions = createAsyncAction(
 
 export const fetchGenomeData: ActionCreator<
   ThunkAction<void, any, null, Action<string>>
-> = (genomeId: string) => async (dispatch) => {
+> = (genomeId: string) => async (dispatch, getState: () => RootState) => {
   await Promise.all([
     dispatch(fetchGenomeInfo(genomeId)),
     dispatch(fetchGenomeTrackCategories(genomeId))
   ]);
 
+  const state = getState();
+  const committedSpecies: CommittedItem[] = getEnabledCommittedSpecies(state);
+  const genomeInfo: GenomeInfo | null = getGenomeInfoById(state, genomeId);
+  if (
+    genomeInfo &&
+    !find(
+      committedSpecies,
+      (species: CommittedItem) => species.genome_id === genomeId
```

Comment:
There is a new selector `getCommittedSpeciesById` added to `dev` now. It will do the finding.

Reply:
That's done